packages/wrangler/src/api/startDevWorker/ConfigController.ts

```diff
@@ -14,30 +13,43 @@ type Options = StartDevWorkerOptions;
 export class ConfigController extends Controller<ConfigControllerEventMap> {
 	config?: Options;
 
-	public set(input: Hook<Options, [Readonly<Options> | undefined]>) {
+	public set(input: Options) {
 		const config = unwrapHook(input, this.latest);
```

Comment:
I don't think that `unwrapHook()` needed here. The type suggests that `input` cannot be a function or a Promise.

Reply:
You're right. I forgot to remove that. Not a problem since that function will return input as-is. But addressed by removing in https://github.com/cloudflare/workers-sdk/pull/6145